### PR TITLE
Added 1.24 jobs

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*
Added jobs for 1.24

/hold until patches are at least done

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templater/jobs/utils/utils.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ var releaseBranches = []string{
 	"1-21",
 	"1-22",
 	"1-23",
+	"1-24",
```
Comment (danbudris) [inline templater/jobs/utils/utils.go:21]: having this templater seems like a blessing

Comment (kschumy) [reply templater/jobs/utils/utils.go:21]: It is! Thanks you, @abhay-krishna 🙌

**Pull request merged**